BookingsApi/BookingsApi.IntegrationTests/Api/V1/Hearings/GetHearingsForNotificationTest.cs

```diff
@@ -14,25 +14,25 @@ public async Task should_return_response_when_first_multi_day_is_in_range_of_que
         // Single day
         var singleDayHearingOutsideRange1 = await Hooks.SeedVideoHearing(options =>
         {
-            options.ScheduledDate = DateTime.UtcNow.AddHours(2);
+            options.ScheduledDate = DateTime.UtcNow.Date.AddHours(2);
```

Comment:
is it necessary to change these dates? This is changing the test data from 2 hours from now, to 2am this morning - which is a datetime in the past

Reply:
When I was working between 11pm to midnight the dates weren't lining up in the query. As a result I needed to update the hearing to make sure it stays outside of the range

Reply:
Fair enough!